### ISSUE TITLE
Algo 8180 provisionig

### ISSUE THREAD

**Issue by ringmor** [fusionpbx/fusionpbx#2424]
Algo is looking for the following files

algom{$mac}.conf
algop8180.conf

In .htaccess there are no rewrites for .conf files. nor algo and cusing it not to generate setting files. Please let me know if this can be fixed.

Thank you

**Comment by markjcrane** [comment]
You are likely thinking of Apache web server. We use nginx by default which doesn't use an .htaccess file. Rewrite is done with inside the nginx configuration.

**Comment by ringmor** [comment]
Yes, I made a mistake. We are using nginx. Even inside nginx the rewrite is not there. What would be the rewrite?
